Add more IDs for PCMCIA cards to HostAP

Signed-off-by: Marcin Juszkiewicz <openembedded@hrw.one.pl>
---

Add more IDs for PCMCIA cards to HostAP

Signed-off-by: Marcin Juszkiewicz <openembedded@hrw.one.pl>
---
 drivers/net/wireless/hostap/hostap/hostap_cs.c |    5 +++++
 1 file changed, 5 insertions(+)

Index: linux-2.6.16/drivers/net/wireless/hostap/hostap_cs.c
===================================================================
--- linux-2.6.16.orig/drivers/net/wireless/hostap/hostap_cs.c	2006-04-24 12:41:34.000000000 +0200
+++ linux-2.6.16/drivers/net/wireless/hostap/hostap_cs.c	2006-04-24 12:42:43.946104760 +0200
@@ -876,6 +876,7 @@
 	PCMCIA_DEVICE_MANF_CARD(0x000b, 0x7300),
 	PCMCIA_DEVICE_MANF_CARD(0x0089, 0x0002), /* Intel PRO/Wireless 2011 */
 	PCMCIA_DEVICE_MANF_CARD(0x0101, 0x0777),
+  	PCMCIA_DEVICE_MANF_CARD(0x0104, 0x0001), /* "Socket", "CF+ LP WLAN Card Rev A", "1.00" */
 	PCMCIA_DEVICE_MANF_CARD(0x0126, 0x8000), /* PROXIM RangeLAN-DS/LAN PC CARD */
 	PCMCIA_DEVICE_MANF_CARD(0x0138, 0x0002), /* Compaq WL100 11 Mbps Wireless Adapter */
 	PCMCIA_DEVICE_MANF_CARD(0x01ff, 0x0008),
@@ -883,9 +884,11 @@
 	PCMCIA_DEVICE_MANF_CARD(0x026f, 0x030b),
 	PCMCIA_DEVICE_MANF_CARD(0x0274, 0x1612), /* Linksys WPC11 Version 2.5 */
 	PCMCIA_DEVICE_MANF_CARD(0x0274, 0x1613), /* Linksys WPC11 Version 3 */
+  	PCMCIA_DEVICE_MANF_CARD(0x0274, 0x3301), /* Linksys WCF11 11Mbps 802.11b WLAN Card */
 	PCMCIA_DEVICE_MANF_CARD(0x028a, 0x0002), /* Compaq HNW-100 */
  	PCMCIA_DEVICE_MANF_CARD(0x02aa, 0x0002), /* ASUS SpaceLink WL-100 */
 	PCMCIA_DEVICE_MANF_CARD(0x02d2, 0x0001),
+  	PCMCIA_DEVICE_MANF_CARD(0x1668, 0x0101), /* Actiontec 802CI2 */
 	PCMCIA_DEVICE_MANF_CARD(0x50c2, 0x0001),
 	PCMCIA_DEVICE_MANF_CARD(0x50c2, 0x7300), /* Airvast WN-100 */
 	PCMCIA_DEVICE_MANF_CARD(0x9005, 0x0021),
@@ -893,8 +896,10 @@
 	PCMCIA_DEVICE_MANF_CARD(0xc00f, 0x0000),
 	PCMCIA_DEVICE_MANF_CARD(0xc250, 0x0002), /* EMTAC A2424i 11Mbps WLAN Card */
  	PCMCIA_DEVICE_MANF_CARD(0xd601, 0x0002), /* NETGEAR MA701" */
+  	PCMCIA_DEVICE_MANF_CARD(0xd601, 0x0004), /* Sitecom WL-007 WLAN CF Card */
  	PCMCIA_DEVICE_MANF_CARD(0xd601, 0x0005), /* D-Link DCF660, Sandisk Connect SDWCFB-000, SMC2642W */
 	PCMCIA_DEVICE_MANF_CARD(0xd601, 0x0010), /* SanDisk ConnectPlus OEM (0MB) */
+ 	PCMCIA_DEVICE_MANF_CARD(0xd601, 0x0101), /* SanDisk ConnectPlus with Memory */
 	PCMCIA_DEVICE_MANF_CARD_PROD_ID1(0x0156, 0x0002, "INTERSIL",
 					 0x74c5e40d),
 	PCMCIA_DEVICE_MANF_CARD_PROD_ID1(0x0156, 0x0002, "Intersil",
